### DISCUSSION TITLE
Nametag

### DISCUSSION THREAD

**Discussion by TehJays** [diamond-rip/Eden#174] [Q&A] [answered]
is there a way to change player nametag during match? like in lunar practice, ur nametag i coloured green and the opponent's nametag is coloured red

**Comment by GoodestEnglish** [comment, marked as answer]
> is there a way to change player nametag during match? like in lunar practice, ur nametag i coloured green and the opponent's nametag is coloured red

Eden has this feature already. If you mean change the color, then it is currently not possible.

**Comment by TehJays** [comment]
how do i enable it? i want it to be like lunar practice or mmc.
 e.g. in mmc the opponent's namecolour and ur own namecolour is according to ur rank)

**Reply by GoodestEnglish** [reply]
You would need an external plugin for this like nametagedit. However I wouldn't recommend because there will be no difference between opponent and teammate in team fight.

**Reply by GoodestEnglish** [reply]
Update: This is added to Eden, you can now set it in config.yml
